dockerfiles/dev/Dockerfile

```diff
@@ -26,25 +26,26 @@
 #
 
 # build using the TerraFERMA development environment
-FROM terraferma/dev-env:fenics-2019.1.0
+FROM terraferma/dev-env:fenics-2019.1.0-focal
 MAINTAINER terraferma <terraferma@lists.columbia.edu>
 
 USER root
 WORKDIR /tmp
 
+
 # install terraferma
-ENV TF_PREFIX /usr/local/terraferma
-RUN export PYTHONPATH=$FENICS_PREFIX/lib/python3.6/site-packages && \
+ENV TF_PREFIX=/usr/local/terraferma 
+RUN export PYTHONPATH=$FENICS_PREFIX/lib/python3.8/site-packages && \
     export PETSC_DIR=$PETSC_PREFIX && \
     export PATH=$FENICS_PREFIX/bin:$PATH && \
     export LD_LIBRARY_PATH=$FENICS_PREFIX/lib:$LD_LIBRARY_PATH && \
     export PKG_CONFIG_PATH=$FENICS_PREFIX/lib/pkgconfig:$PKG_CONFIG_PATH && \
     export SPUD_DIR=$SPUD_PREFIX && \
-    git clone -b master-2019.1.0 -q https://github.com/TerraFERMA/TerraFERMA.git && \
+    git clone -b mspieg/master-2019.1.0-update-fixes -q https://github.com/TerraFERMA/TerraFERMA.git && \
     cd TerraFERMA && \
     mkdir build && \
     cd build && \
-    cmake -DCMAKE_INSTALL_PREFIX=${TF_PREFIX} \
+    cmake -DCMAKE_INSTALL_PREFIX=${TF_PREFIX} -DCMAKE_BUILD_TYPE=Release  \
```

Comment:
Pedantic but could this be split up over multiple lines for consistency with the other cmake commands?

Reply:
as you like...but the Release really needs to be there